code/src/components/AddTodo.css

```diff
@@ -0,0 +1,22 @@
+.add-todo-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin: 10px;
+}
+
+.add-button {
+  color: black;
+  background-color: white;
+  border-color: black;
+  border-width: 1px;
+  border-radius: 50px;
+  font-size: 12px;
+}
```

Comment:
If you want to clean the code you can write the border details in one line: 
border: 1px solid black;